### DISCUSSION TITLE
Naming: español o inglés

### DISCUSSION THREAD

**Discussion by MatiasOlivera** [frontendcafe/cmyk-mango#27] [General]
¿En qué idioma preferís nombrar las variables, las funciones, etc? ¿En español o en inglés?

**Comment by leoachear** [comment]
- Inglés.

Mis primeros cambios quedaron en spanglish 😂

**Comment by lucianinadev** [comment]
- Ingles
Me paso igual 😂 Para acostumbrarme preferiria ingles, quizas y dependiendo deba pedir ayuda para dar un nombre mas correcto a las funciones.

**Comment by luciabrumnich** [comment]
Hola! Ambos idiomas me gustan mucho, aunque creo que sería una buena práctica que sea en inglés.

**Comment by DamianPalavecino** [comment]
Yo prefiero el ingles ya que es el que uso en mi práctica habitual

**Comment by MatiasOlivera** [comment]
Vamos a usar inglés :gb:
